packages/frontend/src/styles/components/_Faq.scss

```diff
@@ -3,27 +3,22 @@
   margin: 0 auto;
   line-height: 1.5;
 
-  h1,
-  h2,
-  h3,
-  h4,
-  h5,
-  h6 {
-    margin: 16px 0 0 0;
-  }
-
-  h1 {
-    font-size: var(--font-xl);
-    margin: 16px 0 32px 0;
-  }
-
   h2 {
-    margin: 32px 0;
+    margin-top: 48px;
     font-size: var(--font-l);
+    line-height: 29px;
+
+    &:first-child {
+      margin-top: 0;
+    }
   }
 
   p {
-    margin: 16px 0 0 0;
+    margin-top: 16px;
```

Comment:
We are phasing out scss, use Tailwind instead. If it possible delete this .scss altogether and style everything in tailwind.

Reply:
It's impossible to delete scss because it styles an auto generated code

Reply:
It is possible, we did something similar for definitions. Look at packages/frontend/src/pages/definitions/style.css, it's imported in packages/frontend/src/styles/style.css and uses tailwind.